Modules/_tkinter.c

```diff
@@ -1182,8 +1182,9 @@ fromWideIntObj(PyObject* tkapp, Tcl_Obj *value)
 {
         Tcl_WideInt wideValue;
         if (Tcl_GetWideIntFromObj(Tkapp_Interp(tkapp), value, &wideValue) == TCL_OK) {
-            if (sizeof(wideValue) <= SIZEOF_LONG_LONG)
-                return PyLong_FromLongLong(wideValue);
+            if (sizeof(wideValue) <= SIZEOF_INTMAX_T) {
```

Comment:
I think this condition is always true. I think that it is always true even for `long long`. I added this check only because the support of `long long` was optional.